Files/App.xaml.cs

```diff
@@ -196,12 +196,13 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
         {
             await logWriter.InitializeAsync("debug.log");
             Logger.Info($"App launched. Prelaunch: {e.PrelaunchActivated}");
-
+            
             //start tracking app usage
             SystemInformation.Instance.TrackAppUse(e);
 
             bool canEnablePrelaunch = ApiInformation.IsMethodPresent("Windows.ApplicationModel.Core.CoreApplication", "EnablePrelaunch");
 
+            LoadOtherStuffAsync().ContinueWith(t => Logger.Warn(t.Exception, "Error during LoadOtherStuffAsync()"), TaskContinuationOptions.OnlyOnFaulted);
```

Comment:
Should this be below `await EnsureSettingsAndConfigurationAreBootstrapped();` and in other places too?

Reply:
Good idea